services/requester/requester.go

```diff
@@ -380,6 +387,50 @@ func (e *EVM) EstimateGas(ctx context.Context, data []byte) (uint64, error) {
 	return gasUsed, nil
 }
 
+func (e *EVM) GetCode(
+	ctx context.Context,
+	address common.Address,
+	height uint64,
+) ([]byte, error) {
+	e.logger.Debug().
+		Str("addess", address.Hex()).
+		Str("height", fmt.Sprintf("%d", height)).
+		Msg("get code")
+
+	hexEncodedAddress, err := cadence.NewString(
```

Comment:
I think this is getting repetitive, lets just add a helper function at the bottom of this file something like: `addressToCadenceString`

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/130/commits/3929b93a2a772ee336bef963ed5f2605da62447f